department-of-veterans-affairs/caseflow
# CASEFLOW-977 Organization Assigned "CAVC Correspondence" Tasks should not be completable

Resolves [CASEFLOW-977](https://vajira.max.gov/browse/CASEFLOW-977)

### Description
User no longer has the ability/option to "Mark task complete"

### Acceptance Criteria
- [ ] Given a CAVC Correspondence Task assigned to the Organization, I should not have the option to `Mark Complete`

### Testing Plan
1. Login as a CAVC lit support user
2. Go to http://localhost:3000/organizations/cavc-lit-support?tab=unassignedTab
3. Select an appeal with "Send CAVC-Remand-Processed Letter Task" to view the Case Details page. Note the docket number and UUID so you can find this appeal later.
4. Switch to a user on the Mail Team (e.g. `JOLLY_POSTMAN`)
5. Revisit the appeal and select "+ Add a new task"
6. In the modal dropdown labeled "Select correspondence type" select "CAVC Correspondence", fill in instructions and submit
7. Switch back to a CAVC lit support user and revisit the appeal once again
8. Confirm you do not have the option to "Mark task complete"

### User Facing Changes

BEFORE | AFTER
---|---
![Screen Shot 2021-03-02 at 5 20 10 PM](https://user-images.githubusercontent.com/18618189/109723143-b0a92100-7b7b-11eb-99f3-b2c4b7b93d4b.png) | ![Screen Shot 2021-03-02 at 5 20 50 PM](https://user-images.githubusercontent.com/18618189/109723151-b3a41180-7b7b-11eb-9204-654af7d0ead6.png)
